templates/rabbitmq-3/2/README.md

```diff
@@ -0,0 +1,12 @@
+RabbitMQ 3.6 with easy Rancher clustering
+===
+Provides RabbitMQ image that can scale to a cluster.
+
+The following environment variables are passed to confd in order to set up RabbitMQ's configuration file:
+
+* Partition handling: RabbitMQ's cluster handling setting: default set to autoheal
+* Erlang cookie: cookie to allow nodes communication: default set to defaultcookiepleasechange
```

Comment:
maybe this should even be a 'password' type so people can hit the generate button, and make required=true and default=nil to force people to give it a value that has no default